frontend/src/components/AdminContent/adminContent.styled.js

```diff
@@ -0,0 +1,58 @@
+import { styled as muiStyled } from "@mui/material/styles";
```

Comment:
Not a big deal - but is there a particular reason this is renamed on import?

Reply:
Nope, it's just cause I copy/pasted the import from create-campaign which had it renamed - will change it to not rename